EnglishTranslation-master/ETC.tsv

```diff
@@ -14166,7 +14166,7 @@ ETC_20150804_014167	Enhance a Shield to level 4
 ETC_20150804_014168	Check the Great King Zachariel's Epitaph
 ETC_20150804_014169	Time until the Light Beacon for the Prayer Hall gets activated: 
 ETC_20150804_014170	The Light Beacon at Fatama Oratorium has been activated!
-ETC_20150804_014171	The Light Beacon at Fatama Oratorium has been activated!{nl}The light shines into the Prayer Hall of the Royal Mausoleum Constructor's Church
+ETC_20150804_014171	The Light Beacon at Fatama Oratorium has been activated!{nl}The light shines into the Prayer Hall of the Royal Mausoleum Constructors' Church
```

Comment:
Hm, I just checked. It is currently messed up.

`왕릉 건축자 예배당` = Royal Mausoleum Contructors' Chapel
`왕릉 노역자 숙소` = Royal Mausoleum Workers Lodge

`ETC_20150401_010449	왕릉 노역자 숙소` should be `Royal Mausoleum Workers Lodge`
`ETC_20150428_011234	왕릉 건축자 예배당` should be `Royal Mausoleum Constructors' Chapel`

Good catch, because now a lot of lines with `왕릉 노역자 숙소` in it should be changed to Royal Mausoleum Workers Lodge